src/main/webapp/app/exercises/text/assess/text-assessment-base.component.ts

```diff
@@ -128,7 +129,7 @@ export abstract class TextAssessmentBaseComponent implements OnInit {
         }
     }
 
-    private addTextBlockByIndices(startIndex: number, endIndex: number, submission: TextSubmission, textBlockRefs: TextBlockRef[]): void {
+    private static addTextBlockByIndices(startIndex: number, endIndex: number, submission: TextSubmission, textBlockRefs: TextBlockRef[]): void {
```

Comment:
Just curious but why not leave the function private?

Reply:
This change was suggested by IntelliJ as this function only modifies its parameters and does not access `this`.